frontend/src/libs/components/profile-settings-sidebar/profile-settings-sidebar.tsx

```diff
@@ -0,0 +1,66 @@
+import { IconColor } from '#libs/enums/icon-color.enum.js';
+import { useAppSelector, useCallback, useState } from '#libs/hooks/hooks.js';
+import { type IconName } from '#libs/types/types.js';
+
+import { Icon } from '../components.js';
+import { ProfileSettingsButton } from './profile-settings-button/profile-settings-button.js';
+import styles from './styles.module.scss';
+
+const ProfileSettingsSidebar: React.FC = () => {
+  const userName = useAppSelector((state) => {
+    return state.auth.authenticatedUser?.fullName;
```

Comment:
Cast type here. We know that there is a user for sure, if the user reached this page